source/process/desktop-release-process.md

```diff
@@ -0,0 +1,158 @@
+# Release Process
```

Comment:
Not sure if we should just keep this in https://github.com/mattermost/desktop/blob/master/docs/release-process.md